app/code/Magento/ReviewGraphQl/Model/Resolver/Product/Review/Rating.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ReviewGraphQl\Model\Resolver\Product\Review;
+
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Review\Model\ResourceModel\Rating\Option\Vote\Collection as VoteCollection;
+use Magento\ReviewGraphQl\Model\DataProvider\ReviewRatingsDataProvider;
+
+/**
+ * Review rating resolver
+ */
+class Rating implements ResolverInterface
```

Comment:
I would suggest renaming this resolver as `RatingBreakdown`. The current name does not reflect the purpose on 100%

Reply:
Done ✅